testsuite/MDAnalysisTests/analysis/test_nucleicacids.py

```diff
@@ -44,11 +44,37 @@ def wc_rna(u):
     return WC
 
 
+def test_wc_dist_shape(wc_rna):
+    assert wc_rna.results.pair_distances.shape == (1, 2)
+
+
+def test_wc_dist_results_keys(wc_rna):
+    assert "pair_distances" in wc_rna.results.keys()
+
+
 def test_wc_dist(wc_rna):
     assert_allclose(wc_rna.results.pair_distances[0, 0], 4.3874702, atol=1e-3)
     assert_allclose(wc_rna.results.pair_distances[0, 1], 4.1716404, atol=1e-3)
 
 
+def test_wc_dist_invalid_residue_types(u):
+    with pytest.raises(ValueError, match=r"are not valid nucleic acids"):
```

Comment:
When writing tests to catch exceptions, you as few lines as possible inside the "with" block, ideally 1. This makes it more precise in what it catches and clearer what is getting tested